tests/util.py

```diff
@@ -23,6 +23,11 @@
 PGP_PUB = os.path.join(DATADIR, 'pgp-pub.asc')
 PGP_PUB_KEYRING = os.path.join(DATADIR, 'pgp-pub.gpg')
 PGP_KEY_ID = '281D0DDC6EDD77CF6A8A936C247D3E51CDA08B6B'
+ED25519_PUBLIC_KEY = \
+    'KDurJTccLAQ4AqtSAU8g7aPV25i6oEsy1TS6gh4LdlE='
+ED25519_PRIVATE_KEY = (
+    '2b19zVhdsWKxoUOGP8OTBwQGAGSshWJCpage7Ov+tUcoO6slNxwsBDgCq1IBTyDto9XbmLqgS'
+    + 'zLVNLqCHgt2UQ==')
```

Comment:
You don't need the `+` to concatenate the strings. This is personal preference and probably not consistent in this repo, but I prefer the closing paren on a separate line. So:

```suggestion
ED25519_PRIVATE_KEY = (
    '2b19zVhdsWKxoUOGP8OTBwQGAGSshWJCpage7Ov+tUcoO6slNxwsBDgCq1IBTyDto9XbmLqgS'
    'zLVNLqCHgt2UQ=='
)
```

Reply:
ah that's personal force of habit, I often accidentally concatenate strings together so I have linters yell at me if I do it :sweat_smile: fixed!